### ISSUE TITLE
search_movie() gets HTTPError 308 Critical error

### ISSUE THREAD

**Issue by haar-making** [cinemagoer/cinemagoer#431]
#### Issue description

Started using Cinemagoer today following examples from the docs.

search_movie() returns a critical HTTPError 308

#### Version of Cinemagoer, Python and OS

**NOTICE: please always try the latest version from the repository before submitting a bug.**

Python: 3.10.9
Cinemagoer: 2022.12.04
OS: system='Windows', node='DESKTOP-D4LT0AJ', release='10', version='10.0.22000', machine='AMD64'

#### Steps to reproduce the issue

*if possible, provide a minimal code to reproduce the problem*

```python
#!/usr/bin/env python

from imdb import Cinemagoer, IMDbError

try:
    ia = Cinemagoer()
    movies = ia.search_movie('rock')
    print(movies[0])
except IMDbError as e:
    print("exception here")
    print(e)
```

#### What's the expected result?
Return an iterable of results that match or are similar to 'rock'.

#### What's the actual result?
2022-12-23 17:21:40,395 CRITICAL [imdbpy] C:\Users\jason\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\imdb\_exceptions.py:32: IMDbDataAccessError exception raised; args: ({'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/find?q=rock&s=tt', 'proxy': '', 'exception type': 'IOError', 'original exception': <HTTPError 308: 'Permanent Redirect'>},); kwds: {}
Traceback (most recent call last):
  File "C:\Users\jason\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\imdb\parser\http\__init__.py", line 222, in retrieve_unicode
    response = uopener.open(url)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 525, in open
    response = meth(req, response)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 634, in http_response
    response = self.parent.error(
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 563, in error
    return self._call_chain(*args)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 496, in _call_chain
    result = func(*args)
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\urllib\request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 308: Permanent Redirect
exception here
{'errcode': None, 'errmsg': 'None', 'url': 'https://www.imdb.com/find?q=rock&s=tt', 'proxy': '', 'exception type': 'IOError', 'original exception': <HTTPError 308: 'Permanent Redirect'>} 

#### Additional details
None

**Comment by alberanid** [comment]
Please install the repository version; this problem was fixed yesterday in #426 

I will release a new version next week.

**Comment by haar-making** [comment]
Superb - thank you!
